File: visa-dev-samplecode-master/vdp-php/tests/vctc/ConsumerRulesTest.php
<?php

namespace Vdp;

class ConsumerRulesTest extends \PHPUnit_Framework_TestCase {
	
	public function setUp() {
		$this->conf = parse_ini_file ( "configuration.ini", true );
		$this->visaAPIClient = new VisaAPIClient;
		$this->cardRegisterData = json_encode ([
		  "primaryAccountNumber" => $this->conf ['VDP'] ['vctcTestPan']
	]);
	}
	
	public function testRegisterACard() {
		$baseUrl = "vctc/";
		$resourcePath = "customerrules/v1/consumertransactioncontrols";
		$statusCode = $this->visaAPIClient->doMutualAuthCall ( 'post', $baseUrl.$resourcePath, 'Register a card call', $this->cardRegisterData );
		$this->assertTrue(($statusCode == "200" || $statusCode == "201"));
	}
}

File: visa-dev-samplecode-master/vdp-c-sharp/vdp-c-sharp/VDPApiTest/vctc/ConsumerRulesTest.cs
using System.Configuration;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace Vdp
{
    [TestClass]
    public class ConsumerRulesTest
    {
        private string cardRegisterData;
        private VisaAPIClient visaAPIClient;

        public ConsumerRulesTest()
        {
            visaAPIClient = new VisaAPIClient();
            cardRegisterData =
                  "{"
                     + "\"primaryAccountNumber\": \"" + ConfigurationManager.AppSettings["vctcTestPan"] + "\""
                + "}";
        }

        [TestMethod]
        public void TestRegisterACard()
        {
            string baseUri = "vctc/";
            string resourcePath = "customerrules/v1/consumertransactioncontrols";
            string status = visaAPIClient.DoMutualAuthCall(baseUri + resourcePath, "POST", "Card Registration Test", cardRegisterData);
            Assert.IsTrue((status.Equals("OK")||status.Equals("Created")));
        }
    }
}
